monailabel/scribbles/utils.py

```diff
@@ -109,22 +109,46 @@ def make_iseg_unary(
     return unary_term
 
 
-def make_histograms(image, scrib, scribbles_bg_label, scribbles_fg_label, bins=32):
+def make_histograms(image, scrib, scribbles_bg_label, scribbles_fg_label, alpha=1, bins=32):
+    # alpha forms the psuedo-counts for Dirichlet distribution used here as
```

Comment:
Type in "psuedo-counts"

Reply:
fixed the typo : psuedo-counts --> pseudo-counts